app/source/css/components/PageMenu.styl

```diff
@@ -37,19 +37,18 @@
           font-size: 15px;
         }
 
-          &:hover {
-             border-bottom: 4px solid white;
-             padding-bottom: 14px !important;
-           }
-             
-           &.active {
-             a {
-               border-bottom: 4px solid;
-               padding-bottom: 14px !important;
-               outline: none;
-             }  
-           } 
-      } 
+        hover() {
+          border-bottom: 4px solid white;
+          padding-bottom: 14px !important;
```

Comment:
I didn't have a chance to check this, but I wonder if this `! important` is really needed? In my projects I generally try to avoid using these unless it's a state change as in `is-open` or some sort of print stylesheet "must-have". Noticing a lot of these in the code base 😛